swagger.yaml

```diff
@@ -3588,6 +3588,14 @@ definitions:
         $ref: '#/definitions/operators'
         x-go-custom-tag: gorm:"embedded;embedded_prefix:operators_"
         x-nullable: true
+      kubeAPIName:
```

Comment:
it should be internal data under https://github.com/openshift/assisted-service/blob/master/internal/common/db.go